src/components/xform-object-button.js

```diff
@@ -16,12 +21,12 @@ const v2 = new THREE.Vector3();
 const q = new THREE.Quaternion();
 const q2 = new THREE.Quaternion();
 
-AFRAME.registerComponent("rotate-button", {
+AFRAME.registerComponent("xform-button", {
```

Comment:
Would prefer `transform` to `xform`. We would have called `xforms` in the `userinput` system `transforms` or `transformFunctions` but we didn't want to risk confusion with the "position/rotation/scale" interpretation of "transform". Here that IS what we mean, so I think it fits better.

Reply:
ah good point. i was trying to save characters since this term is used in many places, but i agree the conflict with the user input concept makes it worth changing.